codelists/actions.py

```diff
@@ -210,20 +209,19 @@ def create_version_with_codes(
     hierarchy=None,
     codeset=None,
     author=None,
+    force_create=False,
 ):
     """Create a new version of a codelist with given codes.
-
     Returns the new version, or None if no version is created because the codes are the
     same as the previous version.  (Future work could allow a force_create parameter.)
-
     `hierarchy` and `codeset` may be passed in if they have already been calculated.
     """
 
     if not codes:
         raise ValueError("No codes")
 
     prev_clv = codelist.versions.order_by("id").last()
-    if set(codes) == set(prev_clv.codes):
+    if set(codes) == set(prev_clv.codes) and force_create is False:
```

Comment:
Minor nit - you could avoid having to do the set comparison if `force_create` is `True`, since in that case we know we're going to have to create a new version anyway

```
if force_create is True or set(codes) != set(prev_clv.codes):
    return _create_version_with_codes(...)
return None
```